<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Listener Suite" duration-ms="28448" started-at="2020-05-30T00:49:17 EDT" finished-at="2020-05-30T00:49:46 EDT">
    <groups>
    </groups>
    <test name="Listener Test" duration-ms="28448" started-at="2020-05-30T00:49:17 EDT" finished-at="2020-05-30T00:49:46 EDT">
      <class name="com.class04.ListenerDemo">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.class04.ListenerDemo@10bbd20a]" name="setUp" is-config="true" duration-ms="17959" started-at="2020-05-30T00:49:17 EDT" finished-at="2020-05-30T00:49:35 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="test1()[pri:0, instance:com.class04.ListenerDemo@10bbd20a]" name="test1" duration-ms="38" started-at="2020-05-30T00:49:35 EDT" finished-at="2020-05-30T00:49:35 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.class04.ListenerDemo@10bbd20a]" name="tearDown" is-config="true" duration-ms="328" started-at="2020-05-30T00:49:36 EDT" finished-at="2020-05-30T00:49:37 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.class04.ListenerDemo@10bbd20a]" name="setUp" is-config="true" duration-ms="8122" started-at="2020-05-30T00:49:37 EDT" finished-at="2020-05-30T00:49:45 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="test2()[pri:0, instance:com.class04.ListenerDemo@10bbd20a]" name="test2" duration-ms="4" started-at="2020-05-30T00:49:45 EDT" finished-at="2020-05-30T00:49:45 EDT">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at com.class04.ListenerDemo.test2(ListenerDemo.java:29)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.class04.ListenerDemo@10bbd20a]" name="tearDown" is-config="true" duration-ms="350" started-at="2020-05-30T00:49:45 EDT" finished-at="2020-05-30T00:49:46 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.class04.ListenerDemo -->
    </test> <!-- Listener Test -->
  </suite> <!-- Listener Suite -->
</testng-results>
